open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -38,18 +38,31 @@ the scope and readiness of a package to be reviewed, they may opt to assign an e
 
 ## Editor in Chief checklist
 
-When a new package is submitted for review, the EiC will:
+When a new package is submitted for review, the Editor in Chief will:
 
-- Tag the issue with `1/editor-checks` tag 
-- Next, they will use the {ref}`template <editor-checklist-template>` below in the issue to check whether the package has 
+### 1. ✅ Tag the issue with `1/editor-checks` tag in GitHub
+
+### 2. ✅ Next, they will use the {ref}`template <editor-checklist-template>` below in the issue 
+
+This step serves to check whether the package has 
 the bare minimum requirements to initiate a review (or they will assign that task to an editor as stated above). 
-- Finally they will ensure the onboarding survey is filled out. Thank the authors graciously for doing this. They can skip sections of it if they wish, but we do need their contact information and we do want to track their experience with our review process and organization.  
+These checks ensure that the package is ready to be reviewed.
+
+Following this step will ensure that we are using our volunteer reviewer time effectively.
+
+In some situations, these checks may be passed down to an editor including:
+ 
+* If the Editor in Chief is overwhelmed with package submissions to evaluate 
```

Comment:
Maybe this can sound a bit more like the two roles work together rather than all of the onus on the EiC. @NickleDave i don't want you or any EiC to feel a huge weight in this role. i want you to feel like a part of a team but you happen to be the first critical step in the process of deciding what packages get a review / what packages are ready for a review.